test/integration/firebolt.dbtspec

```diff
@@ -6,15 +6,16 @@ target:
   engine_name: "{{ env_var('ENGINE_NAME') }}"
   user: "{{ env_var('USER_NAME') }}"
   password: "{{ env_var('PASSWORD') }}"
+  schema: test
   port: 443
   threads: 1
 sequences:
   test_dbt_empty: empty
-  test_dbt_base: base
-  test_dbt_ephemeral: ephemeral
-  test_dbt_incremental: incremental
-  test_dbt_snapshot_strategy_timestamp: snapshot_strategy_timestamp
-  test_dbt_snapshot_strategy_check_cols: snapshot_strategy_check_cols
+#  test_dbt_base: base
```

Comment:
Is this a temporary comment or will these tests not be run?

Reply:
This is temporary, but with a long time frame to incorporation. Some of these features we don't yet support (incremental, ephemeral), some just weren't completing. Thus far we've been relying on Jaffle Shop as a test-bed for PRs, merges, and releases, thus this integration test suite has been superfluous. @stepansergeevitch is currently working to incorporate those that we currently support, and over the course of the next few quarters all the tests will be incorporated, as they're supported by Firebolt. In the  meantime, they were deemed of insufficient importance to include now.

And yeah, I realize that's less than ideal. :-\